docs/pages/routing/appearance.mdx

```diff
@@ -95,6 +95,30 @@ export default function App() {
 }
 ```
 
+## React Navigation themes
+
+React Navigation navigators `<Stack>`, `<Drawer>` and `<Tabs>` use a shared appearance provider. In React Navigation, you set the theme for the entire app using the `<NavigationContainer />` component. Expo Router manages the root container for you, so instead you should set the theme using the `ThemeProvider` directly.
```

Comment:
```suggestion
React Navigation navigators `<Stack>`, `<Drawer>`, and `<Tabs>` use a shared appearance provider. In React Navigation, you set the theme for the entire app using the `<NavigationContainer />` component. Expo Router manages the root container so that you can set the theme using the `ThemeProvider` directly.
```